src/tiled/tilesetdock.h

```diff
@@ -165,7 +165,7 @@ private slots:
     TilesetDocumentsFilterModel *mTilesetDocumentsFilterModel;
 
     QTabBar *mTabBar;
-    QStackedWidget *mViewStack;
+    QStackedWidget *mSuperViewStack, *mViewStack;
```

Comment:
Please define each variable in a new line.

Reply:
Will do on the third commit.